### PR TITLE
created new pageType: amp-story-page

### PR DESCRIPTION
**Pull request by ags1773**

### CODE REVIEW

File: test/amp_tags_test.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const {getSeoMetadata, assertContains} = require("./utils");
 
 const assert = require('assert');
 
-describe('ImageTags', function() {
+describe('AmpTags', function() {
```
Comment (ags1773) [inline test/amp_tags_test.js:6]: Corrected typo

File: src/author-tags.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ import {get} from 'lodash';
  * @param {...*} params See {@link Generator} for other Parameters
  */
 export function AuthorTags(seoConfig, config, pageType, data, {url}) {
-  if(pageType != 'story-page')
+  if(pageType != 'story-page' || pageType != 'story-page-amp')
```
Comment (sharangj) [inline src/author-tags.js:11]: This is not a scalable model. Better make the pageType as `amp-page`.

File: src/structured-data/structured-data-tags.js [hunk 3]
```diff
@@ -221,6 +221,7 @@ function generateBreadcrumbListData(pageType = "", publisherConfig = {}, data =
   switch (pageType) {
     case "section-page": breadcrumbsDataList = breadcrumbsDataList.concat(getSectionPageCrumbs(data.section)); break;
     case "story-page": breadcrumbsDataList = breadcrumbsDataList.concat(getStoryPageCrumbs(data.story)); break;
+    case "story-page-amp": breadcrumbsDataList = breadcrumbsDataList.concat(getStoryPageCrumbs(data.story)); break;
```
Comment (sharangj) [inline src/structured-data/structured-data-tags.js:224]: You can use `breadcrumbs` directly in the data.

File: src/author-tags.js (hunk 2 above)
Comment (ags1773) [reply src/author-tags.js:11]: named it `story-page-amp` because thought we might later want to add more pageTypes for amp section, home pages etc.

**Comment by ags1773** [comment]
image

**Pull request merged**